use-cases/predictive_maintenance/2_dataprep_predmaint.ipynb

```diff
@@ -4,114 +4,85 @@
    "cell_type": "markdown",
```

Comment:
<p>The histogram matrix is a nice visualization - consider adding it to the notebook as a PNG file given that the output is being removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Added, see cell below</p>